Makefile

```diff
@@ -971,7 +971,9 @@ $(LIB_WHISPER): \
 	$(CXX) $(CXXFLAGS) -shared -fPIC -o $@ $^ $(LDFLAGS)
 
 $(LIB_WHISPER_S): \
-	$(OBJ_WHISPER)
+	$(OBJ_WHISPER) \
+	$(OBJ_GGML) \
+	$(LIB_GGML)
```

Comment:
Is this change necessary?

Reply:
ya, otherwise it can't find `ggml_xxx` library

error message like:

```
/usr/bin/ld: whisper.cpp:(.text+0x2a82b): undefined reference to `ggml_time_us'
/usr/bin/ld: whisper.cpp:(.text+0x2a881): undefined reference to `ggml_time_us'
/usr/bin/ld: whisper.cpp:(.text+0x2b191): undefined reference to `ggml_time_us'
/usr/bin/ld: whisper.cpp:(.text+0x2b2e1): undefined reference to `ggml_time_us'
/usr/bin/ld: /home/ubuntu/xxxx/libwhisper.a(whisper.o):whisper.cpp:(.text+0x2b5fe): more undefined references to `ggml_time_us' follow

```